webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -80,9 +75,8 @@ const createOrgItem = orgs => {
   orgs.forEach(org => {
     const childObject = {
       type: org.type,
-      name: isEmpty(org.title) ? org.title : __(org.title),
```

Comment:
not directly related, but this is a mistake, taxonomy titles shouldn't be translated